lecli/lecli.py

```diff
@@ -4,9 +4,10 @@
 import queryapi
 import userapi
 
-
+VERSION = 0.1
```

Comment:
I think it is better if we follow PEP8 naming convention for versioning: http://stackoverflow.com/a/45918